### PR TITLE
Added storyset for only mobile device in the hero section

### PR DESCRIPTION
**Pull request by miyurugunarathna**

### CODE REVIEW

**Review by rusiruavb — APPROVED** [review]
LGTM

File: src/styles/home.scss [hunk 1]
```diff
@@ -133,8 +133,56 @@
...
+  }
+}
+
 .home-3d-model {
-  min-height: 250px;
+  @include media("<=phone") {
+    display: none;
+  }
+
+  @include media(">=phone", '<tablet') {
```
Comment (senuravihanjayadeva) [inline src/styles/home.scss:167]: @include media(">=phone", '>=largedesktop')

i think instead of caparing all breakpoints u can do something like this

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

File: src/styles/home.scss [hunk 2]
```diff
@@ -133,8 +133,56 @@
...
+  }
+
+  @include media('>=largedesktop') {
+    display: none;
+  }
+}
+
 .home-3d-model {
-  min-height: 250px;
+  @include media("<=phone") {
```
Comment (rusiruavb) [inline src/styles/home.scss:163]:
```
.home-3d-model {
  display: block;

  @include media("<=phone") {
    display: none;
  }
}
```

File: src/styles/home.scss [hunk 3]
```diff
@@ -133,8 +133,56 @@
   }
 }
 
+.home-header-story {
+  @include media("<=phone") {
+    display: block;
+  }
+
+  @include media(">=phone", '<tablet') {
```
Comment (senuravihanjayadeva) [inline src/styles/home.scss:141]: @include media(">=phone")

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Review by rusiruavb — CHANGES_REQUESTED** [review]

File: src/styles/home.scss (hunk 3 above)
Comment (senuravihanjayadeva) [reply src/styles/home.scss:141]: this is enogh no need to compare other breapoints.

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**